dev/io.openliberty.org.jboss.resteasy.common/src/org/jboss/resteasy/client/jaxrs/engines/ManualClosingApacheHttpClient43Engine.java

```diff
@@ -0,0 +1,761 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
```

Comment:
We can't add this.  RESTEasy is licensed with Apache v2 and presumably copyrighted by RedHat, so we cannot change the license to EPL or add an IBM copyright to it.

Guess who just finished his Open Source Education! ;-)

Reply:
I was following an example from a trusted source ;-). I'll fix it